src/App.tsx

```diff
@@ -1,61 +1,126 @@
+import { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList/TodoList';
+import todosList from './api/todos';
+import users from './api/users';
+
+function getUserById(userId: number) {
+  return users.find(user => user.id === userId) || null;
+}
+
+export const todos = todosList.map(todo => ({
+  ...todo,
+  user: getUserById(todo.userId),
+}));
 
 export const App = () => {
+  const [todoTitle, setTodoTitle] = useState('');
+  const [selectedUser, setSelectedUser] = useState('');
```

Comment:
The `selectedUser` state is initialized with an empty string, but the `defaultValue` in the select element is set to `0`. This mismatch can cause issues when checking for a selected user. Consider initializing `selectedUser` with `'0'` to match the `defaultValue`.